quinn-proto/src/lib.rs

```diff
@@ -235,3 +235,4 @@ const MIN_MTU: u16 = 1232;
 const TIMER_GRANULARITY: Duration = Duration::from_millis(1);
 /// Maximum number of streams that can be uniquely identified by a stream ID
 const MAX_STREAM_COUNT: u64 = 1 << 60;
+const MAX_ADDITIONAL_DATA_SIZE: usize = 39; // max(ipv4, ipv6) + port + retry_src_cid
```

Comment:
This is specific to the internals of `RetryToken` and hence shouldn't be defined cratewide.